packages/rum/src/getActionNameFromElement.ts

```diff
@@ -1,16 +1,48 @@
 export function getActionNameFromElement(element: Element): string {
   // Proceed to get the action name in two steps:
-  // * first, use strategies that are known to return good results. Those strategies will be used on
+  // * first, get the name programmatically, explicitely defined by the user.
+  // * then, use strategies that are known to return good results. Those strategies will be used on
```

Comment:
what do you think of renaming "getters" to "strategies" in order to have a more consistent wording between comments and implementation?

Reply:
Ok I'll rename this